minvws/nl-kat-coordination
# Upsert report recipe in REST API

### Changes

This changes the POST implementation of the report recipe REST API so that it functions as upsert. This means that if the recipe ID already exists it will update the already existing recipe.

### QA notes

Send POST requests to the API endpoint with an already existing ID and verify that the existing recipe is updated. 

Note: #3746

---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.
